py3status/modules/weather_owm.py

```diff
@@ -0,0 +1,615 @@
+   Available in: format_temp
+ - {humidity}: Contains the formatted result of format_humidity.
+   Available in: format, forecast_format
+ - {humid}: The humidity percentage
+   Available in: format_humidity
+ - {icon}: The icon associated with a formatting section
+   Available in: format, forecast_format, format_clouds, format_rain,
+     format_snow, format_wind, format_humidity, format_pressure,
+     format_temp, format_sunrise, format_sunset
+ - {in}: Measurement in inches.
+   Available in: format_rain, format_snow
+ - {k_max}: The maximum Kelvin temperature.
+   Available in: format_temp
+ - {k_min}: The minimum Kelvin temperature.
+   Available in: format_temp
+ - {kmh_gust}: The speed in km/hr of current wind gusts.
+   Available in: format_wind
+ - {kmh_speed}: The speed in km/hr of current wind speeds.
+   Available in: format_wind
+ - {k}: Current Kelvin temperature reading.
+   Available in: format_temp
+ - {mm}: Measurement in millimeters.
+   Available in: format_rain, format_snow
+ - {mph_gust}: The speed in mph of current wind gusts.
+   Available in: format_wind
+ - {mph_speed}: The speed in mph of current wind speeds.
+   Available in: format_wind
+ - {msec_gust}: The speed in m/sec of current wind gusts.
+   Available in: format_wind
+ - {msec_speed}: The speed in m/sec of current wind speeds.
+   Available in: format_wind
+ - {m}: Measurement in meters.
+   Available in: format_rain, format_snow
+ - {pressure}: Contains the formatted result of format_pressure.
+   Available in: format, forecast_format
+ - {press}: The measurement of current atmospheric pressure in Pascals
+   Available in: format_pressure
+ - {press}: The measurement of current atmospheric pressure at sea-level in
+   Pascals.
+   Available in: format_pressure
+ - {snow}: Contains the formatted result of format_snow.
+   Available in: format, forecast_format
+ - {strftime}: Contains the formatted value of the time according to
+   a specific format string
+   Available in: format_sunrise, format_sunset
+ - {sunrise}: Contains the formatted result of format_sunrise.
+   Available in: format, forecast_format
+ - {sunset}: Contains the formatted result of format_sunset.
+   Available in: format, forecast_format
+ - {temp}: Contains the formatted result of format_temp.
+   Available in: format, forecast_format
+ - {wind}: Contains the formatted result of format_wind.
+   Available in: format, forecast_format
+ - {yrd}: Measurement in yards.
+   Available in: format_rain, format_snow
+
+Example configuration:
+```
+weather_owm {
+  api_key = '...'
+
+  icons {
+    i200 = "☔"
+    i230_i232 = "🌧"
+
+    clouds = "☁"
+  }
+
+  format = '{icon}: {temp}, {forecast}'
+  forecast_num = 3
+}
+```
+Outputs: 🌫: ○: 59°, ⛅ ☼ 🌧`
+- Currently foggy, 59° F outside, with forecast of cloudy tomorrow, sunny the
+  next day, then rainy
+
+Requires:
+  pyowm: A python package for talking with the OpenWeatherMap service
+
+@author alexoneill
+@licence MIT
+'''
```

Comment:
Missing SAMPLE outputs for screenshot. Ignore this if you like to.

PURPOSE: py3status is currently pending to use new docs + screenshots _very soon_ (tm).